packages/appeals-service-api/Dockerfile

```diff
@@ -11,14 +12,15 @@ COPY packages/appeals-service-api/data/lpa-list.csv ./data/lpa-list.csv
 COPY packages/appeals-service-api/api ./api
 COPY packages/appeals-service-api/src ./src
 
-RUN npm ci
+RUN npm ci --omit=dev
+RUN ls
```

Comment:
This is in use.  Previously we were building all of the dev dependencies and then pruning them everywhere, so far the only use case I've seen for building the dev dependencies as part of the docker image is in the forms web app because we use patch package to patch a mongo function.
Specifically the reason that made me look at this change was that the appeals service api uses test containers, which relies on node gyp which relies on python 3.8 or lower so the only way to make it work would've been to add in the python dependencies to the docker build, which just seemed messy seeing as actually the dev dependencies aren't required at all for this build.

Reply:
Yes think it makes sense to only build what's needed, rather than pruning after.

I just meant the 'run ls' line though

Reply:
Oh yeah lol, I'll get rid of that. Didn't even spot it sorry